snapcraft/internal/sources/errors.py

```diff
@@ -23,3 +23,12 @@ class IncompatibleOptionsError(errors.SnapcraftError):
 
     def __init__(self, message):
         super().__init__(message=message)
+
+
+class DigestDoesNotMatchError(errors.SnapcraftError):
+
+    fmt = 'Expected the digest for source to be {expected}, '\
```

Comment:
no need for the \ in there. You can just continue strings in the following line.